src/zope/testrunner/tests/test_doctest.py

```diff
@@ -151,13 +159,15 @@
 
         # omit traceback entries for unittest.py or doctest.py (and
         # their package variants) from output:
-        (re.compile(r'^ +File "[^\n]*(doctest|unittest|case)(/__init__)?.py", [^\n]+\n[^\n]+\n',
+        (re.compile(r'^ +File "[^\n]*(doctest|unittest|case)(/__init__)?.py", '
+                    r'[^\n]+\n[^\n]+\n',
                     re.MULTILINE),
          r''),
-        (re.compile(r'^{\w+} +File "{\w+}[^\n]*(doctest|unittest|case)(/__init__)?.py{\w+}", [^\n]+\n[^\n]+\n',
+        (re.compile(r'^{\w+} +File "{\w+}[^\n]*(doctest|unittest|case)'
+                    r'(/__init__)?.py{\w+}", [^\n]+\n[^\n]+\n',
                     re.MULTILINE),
          r''),
-        (re.compile('import pdb; pdb'), 'Pdb()'), # Py 2.3
+        (re.compile('import pdb; pdb'), 'Pdb()'),  # Py 2.3
```

Comment:
Really Python 2.3?

Reply:
Well, quite - I noticed that, but didn't think it would be right to change it in this already packed PR.